pkg/config/filter.go

```diff
@@ -29,6 +30,26 @@ func (f Filters) Merge(f2 Filters) {
 	}
 }
 
+type FilterFailed struct{
+	Filter string //?
+	Fatal bool
+	Err error
+}
+
+func (f *FilterFailed) Error() string {
+	return fmt.Sprintf("filter: %v isFatal?: %t err: %v", f.Filter, f.Fatal, f.Err)
+}
```

Comment:
The .toString() for the error,  
I didn't know what a proper separator would be so I opted for a space